src/main/python/trec-covid/generate_round3_baselines.py

```diff
@@ -118,103 +189,58 @@ def prepare_final_submissions(qrels):
 
     run3 = 'anserini.final-r3.rf.txt'
     print(f'Generating {run3}')
     os.system(f'python tools/scripts/filter_run_with_qrels.py --discard --qrels {qrels} ' +
               f'--input runs/anserini.covid-r3.abstract.qdel.bm25+rm3Rf.txt --output runs/{run3} --runtag r3.rf')
+    run3_md5 = compute_md5(f'runs/{run3}')
+    assert run3_md5 == final_runs[run3], f'Error in producing {run3}!'
 
     print('')
-    print(f'{run1} checksum=' + hashlib.md5(open(f'runs/{run1}', 'rb').read()).hexdigest())
-    print(f'{run2} checksum=' + hashlib.md5(open(f'runs/{run2}', 'rb').read()).hexdigest())
-    print(f'{run3} checksum=' + hashlib.md5(open(f'runs/{run3}', 'rb').read()).hexdigest())
-
-
-indexes = ['indexes/lucene-index-cord19-abstract-2020-05-19',
-           'indexes/lucene-index-cord19-full-text-2020-05-19',
-           'indexes/lucene-index-cord19-paragraph-2020-05-19']
-
-stored_runs = {
-    'https://www.dropbox.com/s/g80cqdxud1l06wq/anserini.covid-r3.abstract.qq.bm25.txt?dl=1':
-        'd08d85c87e30d6c4abf54799806d282f',
-    'https://www.dropbox.com/s/sjcnxq7h0a3j3xz/anserini.covid-r3.abstract.qdel.bm25.txt?dl=1':
-        'd552dff90995cd860a5727637f0be4d1',
-    'https://www.dropbox.com/s/4bjx35sgosu0jz0/anserini.covid-r3.full-text.qq.bm25.txt?dl=1':
-        '6c9f4c09d842b887262ca84d61c61a1f',
-    'https://www.dropbox.com/s/mjt7y1ywae784d0/anserini.covid-r3.full-text.qdel.bm25.txt?dl=1':
-        'c5f9db7733c72eea78ece2ade44d3d35',
-    'https://www.dropbox.com/s/qwn7jd8vg2chjik/anserini.covid-r3.paragraph.qq.bm25.txt?dl=1':
-        '872673b3e12c661748d8899f24d3ba48',
-    'https://www.dropbox.com/s/2928i60fj2i09bt/anserini.covid-r3.paragraph.qdel.bm25.txt?dl=1':
-        'c1b966e4c3f387b6810211f339b35852',
-    'https://www.dropbox.com/s/6vk5iohqf81iy8b/anserini.covid-r3.fusion1.txt?dl=1':
-        '61cbd73c6e60ba44f18ce967b5b0e5b3',
-    'https://www.dropbox.com/s/n09595t1eqymkks/anserini.covid-r3.fusion2.txt?dl=1':
-        'd7eabf3dab840104c88de925e918fdab',
-    'https://www.dropbox.com/s/ypoe9tgwef17rak/anserini.final-r3.fusion1.txt?dl=1':
-        'c1caf63a9c3b02f0b12e233112fc79a6',
-    'https://www.dropbox.com/s/uvfrssp6nw2v2jl/anserini.final-r3.fusion2.txt?dl=1':
-        '12679197846ed77306ecb2ca7895b011',
-    'https://www.dropbox.com/s/2wrg7ceaca3n7ac/anserini.final-r3.rf.txt?dl=1':
-        '7192a08c5275b59d5ef18395917ff694',
-    'https://www.dropbox.com/s/ilqgky1tti0zvez/anserini.final-r3.fusion1.post-processed.txt?dl=1':
-        'f7c69c9bff381a847af86e5a8daf7526',
-    'https://www.dropbox.com/s/ue3z6xxxca9krkb/anserini.final-r3.fusion2.post-processed.txt?dl=1':
-        '84c5fd2c7de0a0282266033ac4f27c22',
-    'https://www.dropbox.com/s/95vk831wp1ldnpm/anserini.final-r3.rf.post-processed.txt?dl=1':
-        '3e79099639a9426cb53afe7066239011'
-}
+    print(run1 + ' ' * (35 - len(run1)) + run1_md5)
+    print(run2 + ' ' * (35 - len(run2)) + run2_md5)
+    print(run3 + ' ' * (35 - len(run3)) + run3_md5)
```

Comment:
Perhaps f-string this as `print(f'{run1: <35}{run1_md5}')`

Reply:
Didn't know we could do this, thanks!
This addresses @rodrigonogueira4 's comment above also!